src/app/logic/ComputerPlayer.js

```diff
@@ -1,18 +1,23 @@
 import getRandomInt from '../utils/getRandomInt';
+import { Player } from './Player';
+import isAnswerCorrect from './isAnswerCorrect';
 
-export const ComputerPlayer = () => {
-  const player = {
-    askQuestion(question, callback) {
-      if (!question) throw new Error('There is no question');
-      callback(question);
-    },
+class ComputerPlayer extends Player {
+  getAnswer(answers, correctAnswer, callback) {
+    if (answers.length === 0 || !correctAnswer) {
+      throw new Error('There is no answers or correctAnswer');
+    }
+    const randomAnswer = answers[getRandomInt(0, 3)];
+    if (isAnswerCorrect(randomAnswer, correctAnswer)) {
+      this.correctAnswersCounter += 1;
+    }
+    this.allAnswers.push(randomAnswer);
+    if (typeof callback === 'function') callback(randomAnswer);
```

Comment:
Callback in computer player in unnecessary for now

Reply:
As we talked, I left it just in case, it doesn't have to be used, but yeah, if it won't be used I will remove it with next refactor.